presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -1361,23 +1360,11 @@ private static HiveStorageFormat extractHiveStorageFormat(Table table)
         }
         Map<String, String> columnComment = builder.build();
 
-        return handle -> {
```

Comment:
Is this just a refactoring, or is it part of the fix? It looks like nothing functional changed